src/styles/colors.scss

```diff
@@ -0,0 +1,12 @@
+
+$colorPrimary: #8bc34a;
+$colorPrimaryDark: #5a9216;
+$colorPrimaryLight: #bef67a;
+$colorPrimaryExtraLight:#f2ffab;
+$colorAccent:   #790e8b;
+
+
+$scrollbarTrackBackground: #0000;
```

Comment:
```suggestion
$scrollbarTrackBackground: #000000;
```

Reply:
I would stick to convetion with 6 characters after `#`

Reply:
But I need transparent color, not black.